app/marketing/mails.py

```diff
@@ -1107,6 +1111,18 @@ def new_bounty_request(model):
             from_name=_("No Reply from Gitcoin.co"),
             categories=['admin', 'new_bounty_request'],
         )
+
+        if model.github_org_email != '':
+            to_email = model.github_org_email
+            setup_lang(to_email)
+            send_mail(
```

Comment:
![image](https://user-images.githubusercontent.com/1358093/65888399-d70ad180-e375-11e9-8135-d8f29b906443.png)
This is what I get, the email is correctly set here but then is sent to octavioamuchastegui not oc.tavioamuchastegui

Reply:
![image](https://user-images.githubusercontent.com/1358093/65890519-62399680-e379-11e9-9063-f23052bf9d26.png)
never mind is overwrite when running local.